langs/UA.json

```diff
@@ -0,0 +1,192 @@
+    "playlistStream": "Вам не дозволено додавати потокові відео у свій плейлист.",
+    "playlistPositionNotFound": "Не вдається знайти позицію `{0}` у вашому плейлисті [`{1}`]!",
+    "playlistRemoved": "👋 Видалено **{0}** з плейлиста {1} [`{2}`].",
+    "playlistClear": "Ви успішно очистили свій плейлист [`{0}`].",
+    "playlistView": "Перегляд плейлистів",
+    "playlistViewDesc": "```Ім'я | ID: {0} | {1}\nУсього треків: {2}\nВласник: {3}\nТип: {4}\n```",
+    "playlistViewPermsValue": "📖 Читання: ✓    ✍🏽 Запис: {0}    🗑️ Видалення: {1}",
+    "playlistViewPermsValue2": "📖 Читання: {0}",
+    "playlistViewTrack": "Треки",
+    "playlistViewPage": "Сторінка: {0}/{1} | Загальна тривалість: {2}",
+    "inboxFull": "Вибачте! Поштова скринька {0} переповнена.",
+    "inboxNoMsg": "У вашій поштовій скриньці немає повідомлень.",
+    "invitationSent": "Запрошення надіслано {0}.",
+
+    "notInChannel": "{0}, для використання голосових команд ви маєте перебувати в {1}. Будь ласка, перезайдіть, якщо ви вже в голосі!",
+    "noTrackPlaying": "Зараз немає пісень, які грають.",
+    "noTrackFound": "Пісні з таким запитом не знайдено! Будь ласка, вкажіть дійсне посилання.",
+    "noLinkSupport": "Команда пошуку не підтримує посилання!",
+    "voted": "Ви проголосували!",
+    "missingPerms_pos": "Тільки DJ або адміністратори можуть змінювати позицію.",
+    "missingPerms_mode": "Тільки DJ або адміністратори можуть перемкнути режим циклу.",
+    "missingPerms_queue": "Тільки DJ або адміністратори можуть видаляти треки з черги.",
+    "missingPerms_autoplay": "Тільки DJ або адміністратори можуть вмикати або вимикати режим autoplay!",
+    "missingPerms_function": "Тільки DJ або адміністратори можуть використовувати цю функцію.",
+    "timeFormatError": "Неправильний формат часу. Приклад: 2:42PM або 12:39:31",
+    "lyricsNotFound": "Текст пісні не знайдено. Введіть /lyrics <Назва пісні> <Автор> для пошуку тексту.",
+    "missingTrackInfo": "Деяка інформація про трек відсутня.",
+    "noVoiceChannel": "Голосовий канал не знайдено!",
+
+    "playlistAddError": "Вам не дозволено додавати потокові відео в плейлист!",
+    "playlistAddError2": "Сталася помилка під час додавання треків у плейлист!",
+    "playlistlimited": "Ви досягли ліміту! Ви можете додати тільки {0} пісні до свого плейлиста.",
+    "playlistrepeated": "Такий самий трек уже є у вашому плейлисті!",
+    "playlistAdded": "❤️ Додано **{0}** до плейлиста користувача {1} [`{2}`]!",
+
+    "playerDropdown": "Виберіть пісню для переходу ...",
+
+    "buttonBack": "Назад",
+    "buttonPause": "Призупинити",
+    "buttonResume": "Продовжити",
+    "buttonSkip": "Вперед",
+    "buttonLeave": "СТОП",
+    "buttonLoop": "Зациклити",
+    "buttonVolumeUp": "Гучність +",
+    "buttonVolumeDown": "Гучність -",
+    "buttonVolumeMute": "Вимкнути звук",
+    "buttonVolumeUnmute": "Увімкнути звук",
+    "buttonAutoPlay": "Autoplay",
+    "buttonShuffle": "Перемішати",
+    "buttonForward": "+10сек",
```

Comment:
I recommend renaming the ‘+10 seconds’ button to ‘Forward’. This change would reflect the fact that users can modify the time format to different values, such as 30 seconds. Furthermore, it would maintain consistency with the ‘Rewind’ button, which is not labeled as ‘-10 seconds’.